composer.json

```diff
@@ -17,10 +17,10 @@
     ],
     "license": "MIT",
     "require": {
-        "php": ">=7.1"
+        "php": ">=8.1"
```

Comment:
PHP 8.0 has still security fixed until end 2023

Reply:
To be fair, I hadn't PHP 8.0 installed on my machine so I went the quickest way. But I think you're right. Even if it's a breaking change we should maximize compatibility. I'll do it.

Reply:
Oh but wait. PHPUnit 10 has a minimum requirement of PHP 8.1. So I can't go lower and maintain compatibility.

Reply:
```suggestion
        "php": "^8.1"
```